packages/tools/fetch-tool/src/fluidFetchSnapshot.ts

```diff
@@ -76,7 +76,7 @@ function fetchBlobs(prefix: string,
                 reused = false;
                 blob = blobCache.get(blobId);
                 if (blob === undefined) {
-                    blob = storage.read(blobId);
+                    blob = storage.readString(blobId);
```

Comment:
This code is reading generic blobs, so it has no clue if it reads binary or not binary data.
It should either stay the same (which is the easiest), or use readBlob() API (which would be better), but then we need to trace where / how blobCache is used and fix code down the stack to work with binary data, not base64 encoded data